docs/wiki/extra-documents/how-to-mine-with-G1-mini-ASIC.md

```diff
@@ -0,0 +1,94 @@
+# Grin G1 Mini Setup Introductions
+
+## Server connection
+
+- Please connect with the Ethernet and power supply as the following shows.
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-1.png" />
+
+
+Caution: The server and PC must be in the same network segment, otherwise the IP information returned by the server cannot be received.
+
+
+<hr/>
+<br/>
+
+## Find the server IP address.
+
+Download the file [Showip.exe](https://gameway.fun/output/api/ipollo/download?fileName=Showip.rar).
+
+Right click the “Show IP.exe” and run it as Administrator.
+
+Press the “IP Report” button on the server for 3-5 seconds and release it.
+
+<img alt="grin tutorial image 1" width="400px" src="../../assets/images/g1-mini-ASIC/G1-tut-2.png" />
+
+The server IP address will be feedback as shown below:
```

Comment:
`The server IP address will be feedback as shown below` -> `The server IP address will be shown as can be seen below:`

Reply:
Corrected.